src/motion_planning/scripts/task_utils.py

```diff
@@ -96,10 +98,10 @@ def transform(origin, destination, odometry=None, pose=None):
     return transformed 
 
 def publish_desired_pose_global(pose):
```

Comment:
These publishing methods should not be in task utils. Every method in task_utils should not require any state, these publishing methods are instance methods of a task.

Reply:
They don't store any state in task utils, the state is stored in TaskState. There are other methods in task utils that rely on global state (such as transform lookups).

I don't think publishing methods should be instance methods of a task, there's nothing task specific about them.